src/pages/library/DataBrowserToggler.js

```diff
@@ -19,18 +19,23 @@ export const DataBrowserPreviewToggler = ({ onChange, catalogShowing }) => {
   }, [
     div({ style: { display: 'flex', flexDirection: 'column' } }, [
       strong(['Preview the new Data Catalog']),
-      label({
-        role: 'link',
-        style: { fontWeight: 'bold', display: 'flex', flexDirection: 'row', alignItems: 'center', marginTop: 6 }
-      }, [
-        h(Switch, {
-          checked: catalogShowing,
-          onLabel: '', offLabel: '',
-          width: 55, height: 25,
-          onChange
-        }),
-        div({ style: { marginLeft: 10 } }, [`New Catalog ${catalogShowing ? 'ON' : 'OFF'}`])
-      ])
+      h(IdContainer, [id => h(Fragment, [
```

Comment:
Is this a common idiom for making this kind of object? If so, is there a way to make a component so we can avoid the code duplication?

Reply:
Yes, this is fairly common. IdContainer is our abstraction to avoid making code duplication. We could potentially pull out a method for it that abstracted the Fragment, but I think everything else is as we typically do this in the terra-ui.